docs/sources/write/shortcodes/index.md

```diff
@@ -296,3 +289,17 @@ This Markdown renders as:
 ```markdown
 {{</* myshortcode */>}}
 ```
+
+## About version inference
+
+Version inference enables the use of absolute paths that resolve correctly, irrespective of version.
+It uses special syntax using angle bracket delimiters like `<GRAFANA VERSION>`.
+
+As a convention, use the name of the target project, with spaces but no hyphens or underscores, all upper-case.
+For example, `grafana` becomes `GRAFANA`, `grafana-cloud` becomes GRAFANA CLOUD.
+
+The special syntax `<SOMETHING VERSION>`, is inferred from the version of the page.
+
+You can override version inference by including additional metadata in the front matter of the file.
+To override the value of `<GRAFANA VERSION>`, set the `GRAFANA VERSION` parameter in the page's front matter.
+For example, with the front matter `GRAFANA VERSION: next`, the shortcode replaces `<GRAFANA VERSION>` with `next` irrespective of the source content version.
```

Comment:
I think phrasing this more as the writer's intent vs what the system does under the hood might be more helpful. This wording probably isn't entirely accurate but something a long the lines of: 
```suggestion
For example, to set the version to `next` irrespective of the source content version, add the following to the front matter: `GRAFANA VERSION: next`.
```